network/src/commonTest/kotlin/com/wire/kalium/api/tools/json/api/user/self/SelfApiTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.wire.kalium.api.tools.json.api.user.self
+
+import com.wire.kalium.api.ApiTest
+import com.wire.kalium.api.tools.json.model.ErrorResponseJson
+import com.wire.kalium.network.api.ErrorResponse
+import com.wire.kalium.network.api.user.self.SelfApi
+import io.ktor.client.call.receive
+import io.ktor.client.features.ClientRequestException
+import io.ktor.http.HttpStatusCode
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+@ExperimentalCoroutinesApi
+class SelfApiTest : ApiTest {
+    @Test
+    fun givenAValidRegisterLogoutRequest_whenCallingTheRegisterLogoutEndpoint_theRequestShouldBeConfiguredCorrectly() =
+        runTest {
```

Comment:
What's `runTest` advantage vs using `runBlocking`. If nothing relevant to jusitfy it, I guess it would be better to keep using the latter one for coherence.

Reply:
```runBlocking``` can't be used in  kmp test aka ```commonTest``` so it got replaced with  ```runTest``` for more info [kotlin coroutines 1.6.0-RC release notes](https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.6.0-RC)